### ISSUE TITLE
undefined is not an object(evaluating 'route.routeName') 

### ISSUE THREAD

**Issue by cbarlow1993** [akveo/kittenTricks#44]
I've cloned this repo, installed the modules and attempted to run `react-native run-ios`.

The app loads in the simulator, and after the splash screen, I get a crash. See image:

<img width="416" alt="screen shot 2017-12-22 at 14 48 15" src="https://user-images.githubusercontent.com/7342008/34302063-364c5c2e-e727-11e7-89d8-5454fbce94a0.png">

Has anyone else experienced this and got a work around?

**Comment by zolotyx** [comment]
@cbarlow1993 , getting the same issue here.

**Comment by zolotyx** [comment]
@cbarlow1993 , please take a look at this [comment](https://github.com/react-community/react-navigation/issues/3148#issuecomment-352778884)

**Comment by cbarlow1993** [comment]
Thanks Zolotyx for the pointer. I’m new to react and this repo. Do you know what file / line numbers I would add those suggested edits on that thread in this repo?

**Comment by cbarlow1993** [comment]
And to answer my own question
 Update app.js file from line 28 to look like this. 
```
const KittenApp = StackNavigator({
  First: {
    screen: Screens.SplashScreen
  },
  Home: {
    screen: DrawerNavigator({
        ...AppRoutes,
      },
      {
        drawerOpenRoute: 'DrawerOpen',
        drawerCloseRoute: 'DrawerClose',
        drawerToggleRoute: 'DrawerToggle',
        contentComponent: (props) => <SideMenu {...props}/>
      },

    )
  },
  
}, {
  headerMode: 'none',
});
```

I'll add a PR for it later.

**Comment by sergey-kozel** [comment]
Hi @cbarlow1993, @zolotyx 

Thank you for your help with this issue.

The fix was applied in 5e999b6212ea8363b9db0a5d90a22ea31ac9e573
